include/watchdog.h

```diff
@@ -1,23 +1,39 @@
 /*
- * Copyright (c) 2015 Intel Corporation.
+ * Copyright (c) 2017 Nordic Semiconductor ASA
```

Comment:
You should not remove existing copyrights, just add the new copyright on top please.